src/show/components/Button/index.js

```diff
@@ -14,10 +14,10 @@ const Button = ({
     secondary: disabled ? `text-white` : `text-white hover:text-secondary-200`,
     outline: disabled
       ? `border-none rounded-full text-white`
-      : `border-2 rounded-full border-primary-50 text-primary-50 hover:border-white hover:bg-primary-50 hover:text-white`,
+      : `border rounded-full border-primary-50 text-primary-50 hover:border-white hover:bg-primary-50 hover:text-white`,
     'invert-outline': disabled
-      ? `border-2 rounded-full border-green-300 text-greeen-300 border-white bg-green-300 py-1 text-white`
-      : `border-2 rounded-full border-green-500 text-greeen-500 border-white bg-green-500 py-1 text-white hover:bg-green-600`
+      ? `border rounded-full border-green-300 text-greeen-300 border-white bg-green-300 py-1 text-white`
+      : `border rounded-full border-green-500 text-greeen-500 border-white bg-green-500 py-1 text-white hover:bg-green-600`
```

Comment:
You have back ticks yet I don't see the need for them here.

Reply:
done